README.md

```diff
@@ -0,0 +1,103 @@
+# `data-${dataset_name}`
+
+Part of [`spinalcordtoolbox`](https://github.com/spinalcordtoolbox).
+
+
+## Using the Template
+
+1. Make a [new repo](https://github.com/new) using this as its template.
+
+    Name it `spinalcordtoolbox/data-${dataset_name}` where `${dataset_name}` is something usefully descriptive:
+
+    ![data-template-new](./.data-template-new.png)
+
+2. Download the new repo
+    
+    ```bash
+    git clone git@github.com:spinalcordtoolbox/data-${dataset_name}.git
+    cd data-${dataset_name}
+    ```
+    
+3. Fill with initial metadata
+    
+    ```bash
+    # 1. Find and replace ${dataset-name} with the name you picked (e.g. `supercats`)
+    perl -pi -e 's/\${dataset_name}/supercats/' README.md
+    perl -pi -e 's/\${dataset_name}/supercats/' setup.py
+   
+    # 2. Rename the repo's `dataset` folder to match the repo name
+    git mv src/spinalcordtoolbox/data/dataset src/spinalcordtoolbox/data/${dataset_name}
+   
+    # 3. Commit the changes 
+    git add -u
+    git commit -m "Setting data-${dataset_name} package metadata"
+    git push
+    ```
+    
+4. Fill with initial data and upload
+    
+    ```bash
+    # 1. Copy over the data files to the dataset folder in this repo
+    cp ${data_files} src/spinalcordtoolbox/data/${dataset_name}
+   
+    # 2. Add, commit, and push the newly-added files
+    git add .
+    git commit -m "Copy over data files to dataset folder"
+    git push
+    ```
+
+5. Remove the "Using this template" section from `README.md`, since it is now no longer needed.
+
+    ```bash
+    # Delete the 'Using the Template' section from `README.md
+    vi README.md
+   
+    # Add, commit, and push the changes
+    git add README.md
+    git commit -m "Removing template section from README.md"
+    git push
+    ```
+
+
+## How to update this dataset
+
+1. Edit and commit the files in `src/spinalcordtoolbox/data/`
+2. Go to https://github.com/spinalcordtoolbox/data-${dataset_name}/releases
+
+    1. Click "Draft Release"
+    2. Fill in a version tag. We use date-based release names (e.g. `r20200101`, `r20220518`, etc.)
```

Comment:
Just realizing now... date-based release names aren't kosher with PyPI, are they? 

I feel like I need to try this on test.pypi.org out before being able to comfortable recommend this.

Reply:
Actually they are. One of the reasons https://peps.python.org/pep-0440/ is so complicated is because it didn't want to offend projects using date-based or even [pi-based](https://en.wikipedia.org/wiki/TeX#TeX82) or even the time-based choice [pytz uses](https://peps.python.org/pep-0440/#olson-database-versioning).

Reply:
![Screenshot 2022-06-03 at 20-39-32 PEP 440 – Version Identification and Dependency Specification peps python org](https://user-images.githubusercontent.com/987487/171969678-108ff15a-b8ba-426a-8bb7-cb3074c69281.png)

(dropping the dots between year and month still works right, the relative ordering is the same)

Reply:
Oh one thing though is the 'r' might not be kosher.

Reply:
Thank you!! :)

Reply:
Yep, all good: https://test.pypi.org/project/spinalcordtoolbox-data-template/2022.6.3/